CHANGELOG.md

```diff
@@ -10,7 +10,7 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
-Work in this release was contributed by @antonis, and @maximepvrt. Thank you for your contributions!
```

Comment:
antonis change went out on v8, so I added it to the release there manually and removed him here.